### DISCUSSION TITLE
Need list of txn

### DISCUSSION THREAD

**Discussion by trgowtham** [codereverser/casparser#32] [Q&A] [answered]
I need list of txn without consolidating them. Is there an option for that?

Something like

date, fund name, txn type, NAV, units, amount ...

I want list of all txns in the statement like above. Please let me know if it is possible.

**Comment by codereverser** [comment, marked as answer]
The csv output exactly does this .. 

It prints the parsed data in the following format as a CSV 
```
amc,folio,pan,scheme,isin,amfi,date,description,amount,units,nav,balance,type,dividend
```

Usage:

_cmd_
```bash
$ casparser -p password -o output.csv filename.pdf

Generating Detailed CSV file...
File saved : output.csv

```

OR _in script_
```python
csv_data = read_cas_pdf(filename, password=passwd, output="csv")
```

Can you please test and let me know?

**Reply by trgowtham** [reply]
Thanks a lot! It is working as expected.
